libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -286,6 +287,27 @@ void AP_AHRS_NavEKF::update_SITL(void)
                                         fdm.zAccel);
         }
         _accel_ef_ekf_blended = _accel_ef_ekf[0];
+
+    }
+    // use SITL states to write body frame odometry data at 20Hz
+    uint32_t timeStamp_ms = AP_HAL::millis();
+    if (timeStamp_ms - _last_body_odm_update_ms > 50) {
+        float quality = 100.0f;
+        Vector3f posOffset = Vector3f(0.0f,0.0f,0.0f);
+        float delTime = 0.001f*(timeStamp_ms - _last_body_odm_update_ms);
```

Comment:
By the way, it's not necessary to set posOffset to a zero vector.  Vector3f does that upon initialisation.